wrappers/fastqc/wrapper.py

```diff
@@ -0,0 +1,34 @@
+__author__ = "Thomas Cokelaer"
+__copyright__ = "Copyright 2021, Sequana Team"
+__email__ = "thomas.cokelaer@pasteur.fr"
+__license__ = "BSD-3"
+
+
+from snakemake.shell import shell
+
+
+# Get rule information (input/output/params...)
+input_fastq = snakemake.input
+params = snakemake.params
+log = snakemake.log[0]
+
+# Note that if the input file is empty, fastqc will fail. 
+# We need to touch  a file in such case and your Snakefile
+# should define as output a "done" file.
+
+if isinstance(input_fastq, str):
+    input_fastq = [input_fastq]
+else:
+    input_fastq = input_fastq
+
+if len(input_fastq) != 0:
+    for this in input_fastq:
+        if this.endswith(".bam") or this.endswith("sam"):
```

Comment:
Je suis pas fan du `this` ca me fait penser a du javascript et tu peux faire `this.endswith((".bam", ".sam"))`

Reply:
fixé